docs/modules/ROOT/pages/user-guide/first-steps.adoc

```diff
@@ -1,98 +1,111 @@
 
 == Prerequisites
 
 * A `git` client
 * An IDE
 * JDK 8+ with JAVA_HOME configured appropriately
 * Apache Maven 3.6.2+
-* GraalVM with `native-image` command installed and `GRAALVM_HOME` environment variable set, see
+* GraalVM with the `native-image` command installed and the `GRAALVM_HOME` environment variable set. See
   https://quarkus.io/guides/building-native-image-guide[Building a native executable] section of the Quarkus
   documentation.
 * If you are on Linux, `docker` is sufficient for the native mode too. Use `-Pnative,docker` instead of `-Pnative`
   if you choose this option.
 
-== Step by step with the `rest-json` example
+== code.quarkus.io
+
+Projects can be bootstrapped and generated at https://code.quarkus.io. All of the Camel Quarkus extensions can be found under the 'Integration' heading. Use the 'search' field to help with finding extensions that you are interested in.
+
+Simply select the component extensions that you want to work with and click the 'Generate your application' button to download a basic skeleton project. There is also the option to push the project directly to GitHub.
+
+When the project archive download has completed successfully, unzip and import into your favorite IDE.
+
+== Maven plugin
+
+Quarkus provides a Maven plugin that enables you to quickly bootstrap projects. For example, to create a project skeleton that includes the `timer` and `log` component extensions:
 
-1. Clone Camel Quarkus and check out the latest release tag
-+
 [source,shell]
 ----
-$ git clone https://github.com/apache/camel-quarkus.git
-$ cd camel-quarkus
-# checkout the latest tag
-$ git checkout $(git describe --abbrev=0)
+$ mvn io.quarkus:quarkus-maven-plugin:1.8.3.Final:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=getting-started \
+    -Dextensions=camel-quarkus-log,camel-quarkus-timer
+
+$ cd getting-started
 ----
 
-2. Copy the `rest-json` example out of the Camel Quarkus source tree.
+NOTE: Windows users should omit the `\` if using `cmd`. When using `Powershell`, wrap the `-D` parameters in double quotes.
+
+
+https://gradle.org/[Gradle] support is also available. See the https://quarkus.io/guides/gradle-tooling[Quarkus Gradle] guide for more information.
+
+== IDE plugins
+
+Quarkus has plugins for most of the popular development IDEs. They provide Quarkus language support, code / config completion, project creation wizards and much more. The plugins are available at each respective IDE marketplace.
+
+* https://marketplace.eclipse.org/content/quarkus-tools[Eclipse plugin] 
+* https://plugins.jetbrains.com/plugin/13234-quarkus-tools[IntelliJ plugin]
+* https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus[VSCode plugin]
+
+Check the plugin documentation to discover how to create projects for your preferred IDE.
+
+== Example projects
+
+Camel Quarkus provides a GitHub repository containing set of example projects.
+
+https://github.com/apache/camel-quarkus-examples
```

Comment:
I wonder whether we should mention the "List of extensions" page in addition to https://github.com/apache/camel-quarkus-examples ? 

https://camel.apache.org/camel-quarkus/latest/user-guide/examples.html 

or `xref:user-guide/examples.adoc`

I think it gives a bit better overview of which examples are there compared to https://github.com/apache/camel-quarkus-examples